timemachine/fe/bar.py

```diff
@@ -132,13 +132,16 @@ def bootstrap_bar(w_F, w_R, n_bootstrap=1000, timeout=10):
 
     t0 = time()
 
+    seed = 2022
+    rng = np.random.RandomState(seed)
```

Comment:
nit: any reason not to use the newer [`Generator` API](https://numpy.org/doc/stable/reference/random/generator.html) instead?
```suggestion
    rng = np.random.default_rng(seed)
```

Reply:
Nope! Updated